doc/monitoring/metrics_reference.rst

```diff
@@ -274,7 +283,9 @@ CPU metrics
 
 Those metrics provides CPU usage statistics. Only for Linux.
 
-* ``tnt_cpu_count`` - the number of processors
+* ``tnt_cpu_count`` - the number of processors configured by the operating system
```

Comment:
I still don't get it clearly. Is it system total number that not related to running Tarantool processes at all?